config/environments/production.rb

```diff
@@ -119,5 +119,15 @@
                  port: Settings.redis.port, drive: :hiredis }
   }
 
+  config.middleware.insert_before 0, 'Rack::Cors', logger: (-> { Rails.logger }) do
+    allow do
+      origins /[a-z0-9]+\.sumofus\.org/Z/i
```

Comment:
Not super familiar with the CORS header format, but if we can use the star in a string, we could have an env var with the value: "*.sumofus.org".

Re: testing, I would add a request spec that tests the header is being setup correctly. But yeah, it's tricky since this is in production.rb. If you really want to test that then I guess we could extract that code outside the production.rb file and dynamically load it in the test env in order to test it.